### PR TITLE
FEATURE: FET-129 Added Prettier support

### PR DESCRIPTION
**Pull request by vrkuvrku**
_Description of changes (if multi-commit, short global summary & context;
if single-commit, feel free to leave empty)._

---

# PR checklist

- [x] Change was tested using dev-release in Analytical Designer and Dashboards (if applicable).
- [x] Change is described in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Migration guide (for major update) is written to [CHANGELOG.md](../blob/master/CHANGELOG.md).

# Related Jira tasks

Example:
- [FET-129](https://jira.intgdc.com/browse/FET-129)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/290/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/346/ : FAILURE in 1m 21s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/291/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/347/ : FAILURE in 1m 08s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/292/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/348/ : SUCCESS in 2m 27s

File: tslint.json [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "@gooddata/tslint-config"
-}
+  "extends": [
+    "@gooddata/tslint-config",
+    "tslint-config-prettier"
+  ]
+}
```
Comment (martinmosko) [inline tslint.json]: end of line

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/293/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/349/ : SUCCESS in 1m 12s

File: tslint.json (hunk 1 above)
Comment (vrkuvrku) [reply tslint.json]: done

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/297/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/353/ : SUCCESS in 1m 15s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/298/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/354/ : SUCCESS in 1m 01s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/79/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/355/ : SUCCESS in 1m 43s

**Pull request merged**